doc/chomp_interface/scripts/collision_scene_example_cluttered.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+
+class CollisionSceneExampleCluttered(object):
+
+    def __init__(self):
+        self._scene = PlanningSceneInterface()
+
+        # clear the scene
+        self._scene.remove_world_object()
+
+        self.robot = RobotCommander()
+
+        # pause to wait for rviz to load        
+        print "============ Waiting while RVIZ displays the scene with four obstacles..."
+        rospy.sleep(4)
+
+        floor_pose = [0, 0, -1.12, 0, 0, 0, 1]
```

Comment:
From here on should be a separate method not part of the init.

Reply:
done